src/core/sensor/qgsiodevicesensor.cpp

```diff
@@ -401,14 +437,15 @@ void QgsSerialPortSensor::handleError( QSerialPort::SerialPortError error )
 bool QgsSerialPortSensor::writePropertiesToElement( QDomElement &element, QDomDocument & ) const
 {
   element.setAttribute( QStringLiteral( "portName" ), mPortName );
+  element.setAttribute( QStringLiteral( "baudrate" ), mBaudrate );
```

Comment:
Since we're going to change this to be a proper enum, we can cast to int to save the value.
```suggestion
  element.setAttribute( QStringLiteral( "baudRate" ), static_cast<int>( mBaudrate ) );
```